### ISSUE TITLE
Add ability to expand issue/PR item to summary

### ISSUE THREAD

**Issue by karlnp** [gittyapp/feedback#29]
I have a ton of issues being commented on constantly for days on end & it's difficult to keep them straight in Gitty, which is otherwise great. If I could get a title and summary option for the timeline view, I'd appreciate it.

**Comment by mps** [comment]
@karlnp - when you say timeline view are you saying the 'News Feed' view?

**Comment by karlnp** [comment]
Yes, sorry.

**Comment by mps** [comment]
You don't want to tap on it to open up issue just to see the body right?

**Comment by karlnp** [comment]
That would work too. I was originally thinking about a config setting to display issue titles alongside the existing information, but anything that lets me know what a given issue is would be nice. Github's own news feed behaviour would work as well, where it quotes the comment or PR body.

**Comment by mps** [comment]
@karlnp - originally I did experiment with the title being added to the issue # but things got a bit wordy.  I will experiment with it and see what I can come up with :)

**Comment by karlnp** [comment]
Thank you!
